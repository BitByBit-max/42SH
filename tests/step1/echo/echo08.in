echo -n -e -x -n -e bad option+-x -n -e bad option