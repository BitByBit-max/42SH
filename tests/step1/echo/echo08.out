-x -n -e bad option+-x -n -e bad option